source/client-side-encryption/client-side-encryption.md

```diff
@@ -537,8 +537,7 @@ a name (e.g. `aws:myname`) do not support automatic credentials. Attempting to c
 automatic credentials results in a runtime error from [libmongocrypt](#libmongocrypt). See
 [Why do on-demand KMS credentials not support named KMS providers?](#why-do-on-demand-kms-credentials-not-support-named-kms-providers)
 
-> \[!NOTE\]
->
+> [!NOTE]
```

Comment:
If you view the rendered output, this isn't actually displaying as an alert. Likewise for the instances below.

Wasn't a problem in `auth.md`, so I'm not sure what the issue might be (other than some of the later instances omitting a line break between `[!NOTE]` and the text that follows.

Reply:
Fixed.  Apparently we have to use `span` instead of `div` for the manual targets, or else GitHub chokes on the admonitions.

Reply:
I only just realized that none of the `<span>` (and `<div>`s before them) have closing tags. That seems problematic.

Did you consider using `<span id="..." />` or `<span id="..."></span>`?

Reply:
Done